### PR TITLE
[Merged by Bors] - Picasso-bonding-code-refactors

### PR DESCRIPTION
**Pull request by easteregg**
## Issue
https://app.clickup.com/t/2rpkbup

## Description
This PR fixes some of the redundant code and overall improves the quality of the code to be more maintainable. 

## Checklist

- [ ] I have updated the cargo docs to reflect changes made by this PR _(if applicable)_
- [ ] I have updated the `book/` to reflect changes made by this PR _(if applicable)_

### CODE REVIEW

**Comment by ComposableFiB0t** [comment]
Task linked: [CU-2rpkbup Picasso - FE - Code improvements](https://app.clickup.com/t/2rpkbup)

**Review by andresmechali — APPROVED** [review]
LGTM

File: frontend/apps/picasso/components/Molecules/MyBondingsTable.tsx [hunk 1]
```diff
@@ -20,7 +20,7 @@ import { ActiveBond } from "@/stores/defi/polkadot/bonds/slice";
 
 export type MyBondingsTableProps = TableContainerProps & {
   onRowClick?: (offerId: string) => void;
-  openPositions: ActiveBond[];
+  items: ActiveBond[];
```
Comment (saadjhk) [inline frontend/apps/picasso/components/Molecules/MyBondingsTable.tsx:23]: items do not provide good context, suggest changing the name here?

File: frontend/apps/picasso/defi/polkadot/hooks/useActiveBonds.ts [hunk 2]
```diff
@@ -1,10 +1,11 @@
 import { BondOffer } from "@/stores/defi/polkadot/bonds/types";
 import { useEffect } from "react";
-import { usePicassoProvider } from "@/defi/polkadot/hooks/index";
+import { usePicassoProvider, useSelectedAccount } from "@/defi/polkadot/hooks/index";
```
Comment (saadjhk) [inline frontend/apps/picasso/defi/polkadot/hooks/useActiveBonds.ts:3]: @easteregg did we have a ticket/task for migration to `substrate-react`

**Review by saadjhk — APPROVED** [review]

File: frontend/apps/picasso/defi/polkadot/hooks/useActiveBonds.ts (hunk 2 above)
Comment (easteregg) [reply frontend/apps/picasso/defi/polkadot/hooks/useActiveBonds.ts:3]: Not at the moment, will create one.

**Review by saadjhktuta — APPROVED** [review]

**Review by saadjhk — APPROVED** [review]

**Review by andresmechali — APPROVED** [review]

**Review by andresmechali — APPROVED** [review]

**Review by saadjhk — APPROVED** [review]

**Pull request closed without merging**